aws-api/src/main/java/com/amplifyframework/api/aws/AWSApiPlugin.java

```diff
@@ -383,13 +388,13 @@ private String getIdentityValue(String identityClaim) throws ApiException {
         return identityValue;
     }
 
-    private ArrayList<String> getUserGroups() throws ApiException {
-        CognitoUserPoolsAuthProvider cognitoProvider = getAuthProvider();
+    private ArrayList<String> getUserGroups(AuthorizationType authType) throws ApiException {
+        // TODO: this doesn't work with OIDC right now. Implement custom groups claim.
```

Comment:
What needs to happen to support this?  

Instead of leaving this TODO here, can we either:

1. Add support for it
2. Or, throw an exception if a customer tries to use it?

Reply:
Will be addressed in a separate PR to support custom group claims

Reply:
Implementing in a separate PR is fine.  To merge this now though, could you just replace the TODO with something like this:
```
if(AuthorizationType.OPENID_CONNECT.equals(authType)) {
    throw new ApiException("Custom group claims with OIDC are not supported yet");
}
```